site-unused/qmd/login.qmd

```diff
@@ -1,29 +1,29 @@
 ---
-title: "Log into the {{< var vm_platform >}}"
+title: "Log into the {{< var validmind_platform >}}"
```

Comment:
FYI only, I plan to resurrect this topic to document the cloud providers and regions we now support for the platform UI.

Reply:
For some reason I can't commit this suggestion? 🤔

Reply:
Oh, it looks like you pushed the change, great.